src/Migrate-From-Self-Hosted-to-EMS.md

```diff
@@ -191,3 +207,78 @@ We strongly recommend that you leave the export and Synapse untouched until the
 ## Note on users and Element
 
 Element does have support for changing the delegated homeserver URL. All your users will have to sign out and sign in again to Element. You should ensure everyone has Key Backup configured and working.
```

Comment:
Add instructions in the prep section on how to best log out all devices before the migration

Reply:
you will add the `how to best logout all devices berfore migration` in a separate PR right?